subprojects/docs/src/docs/userguide/authoring-builds/build_lifecycle.adoc

```diff
@@ -15,37 +15,52 @@
 
 [[sec:build_phases]]
-== Build phases
+== Build Phases
 
 A Gradle build has three distinct phases.
+Gradle runs these phases in order: first initialization, then configuration, and finally execution.
 
 Initialization::
-Gradle supports single and multi-project builds. During the initialization phase, Gradle determines which projects are going to take part in the build, and creates a link:{groovyDslPath}/org.gradle.api.Project.html[Project] instance for each of these projects.
+- Detects the <<organizing_gradle_projects#sec:settings_file,settings file>>.
+- Evaluates the settings file to determine which projects and included builds participate in the build.
+- Creates a link:{groovyDslPath}/org.gradle.api.Project.html[`Project`] instance for every project.
 Configuration::
-During this phase the project objects are configured. The build scripts of _all_ projects which are part of the build are executed.
+- Evaluates the build scripts of every project participating in the build.
+- Creates a task graph for requested tasks.
 Execution::
-Gradle determines the subset of the tasks, created and configured during the configuration phase, to be executed. The subset is determined by the task name arguments passed to the `gradle` command and the current directory. Gradle then executes each of the selected tasks.
-
-
-[[sec:settings_file]]
-== Settings file
+- Schedules and executes each of the selected tasks in the order of their dependencies.
 
-Beside the build script files, Gradle defines a settings file. The settings file is determined by Gradle via a naming convention. The default name for this file is `settings.gradle`. Later in this chapter we explain how Gradle looks for a settings file.
+=== Example
 
-The settings file is executed during the initialization phase. A multi-project build must have a `settings.gradle` file in the root project of the multi-project hierarchy. It is required because the settings file defines which projects are taking part in the multi-project build (see <<multi_project_builds.adoc#multi_project_builds,Authoring Multi-Project Builds>>). For a single-project build, a settings file is optional. Besides defining the included projects, you might need it to add libraries to your build script classpath (see <<organizing_gradle_projects.adoc#organizing_gradle_projects,Organizing Gradle Projects>>). Let's first do some introspection with a single project build:
+The following example shows which parts of settings and build files correspond to various build phases:
 
-.Single project build
 ====
 include::sample[dir="snippets/buildlifecycle/basic/groovy",files="settings.gradle[];build.gradle[]"]
 include::sample[dir="snippets/buildlifecycle/basic/kotlin",files="settings.gradle.kts[];build.gradle.kts[]"]
 ====
 
-=== Output of **`gradle test testBoth`**
 [source.multi-language-sample,groovy]
```

Comment:
```suggestion
The following command configures and executes the `test` and `testBoth` tasks specified above.
Because Gradle only configures requested tasks and their dependencies, the `configured` task never configures.

[source.multi-language-sample,groovy]
```

@eskatos Can you double-check this to ensure technical correctness? https://github.com/gradle/gradle/pull/22455/files#r1000237190